src/sentry/static/sentry/app/views/organizationIntegrations/pluginDetailedView.tsx

```diff
@@ -0,0 +1,340 @@
+} & RouteComponentProps<{orgId: string; pluginSlug: string}, {}>;
+
+class PluginDetailedView extends AsyncComponent<
+  Props & AsyncComponent['props'],
+  State & AsyncComponent['state']
+> {
+  componentDidMount() {
+    const {location} = this.props;
+    const value =
+      location.query.tab === 'configurations' ? 'configurations' : 'information';
+
+    // eslint-disable-next-line react/no-did-mount-set-state
+    this.setState({tab: value});
+  }
+
+  getEndpoints(): ([string, string, any] | [string, string])[] {
+    const {orgId, pluginSlug} = this.props.params;
+    return [
+      ['plugins', `/organizations/${orgId}/plugins/configs/?plugins=${pluginSlug}`],
+    ];
+  }
+  get plugin() {
+    return this.state.plugins[0];
+  }
+
+  handleResetConfiguration = (projectId: string) => {
+    //make a copy of our project list
+    const projectList = this.plugin.projectList.slice();
+    //find the index of the project
+    const index = projectList.findIndex(item => item.projectId === projectId);
+    //should match but quit if it doesn't
+    if (index < 0) {
+      return;
+    }
+    //remove from array
+    projectList.splice(index, 1);
+    //update state
+    this.setState({
+      plugins: [{...this.state.plugins[0], projectList}],
+    });
+  };
+
+  handleEnablePlugin = (projectId: string) => {
+    //make a copy of our project list
+    const projectList = this.plugin.projectList.slice();
+    //find the index of the project
+    const index = projectList.findIndex(item => item.projectId === projectId);
+    //should match but quit if it doesn't
+    if (index < 0) {
+      return;
+    }
+
+    //update item in array
+    projectList[index] = {
+      ...projectList[index],
+      enabled: true,
+    };
+
+    //update state
+    this.setState({
+      plugins: [{...this.state.plugins[0], projectList}],
+    });
+  };
+
+  handleAddToProject = () => {
+    const plugin = this.plugin;
+    const {organization, router} = this.props;
+    openModal(
+      ({closeModal, Header, Body}) => (
+        <ContextPickerModal
+          Header={Header}
+          Body={Body}
+          nextPath={`/settings/${organization.slug}/projects/:projectId/plugins/${plugin.id}/`}
+          needProject
+          needOrg={false}
+          onFinish={path => {
+            closeModal();
+            router.push(path);
+          }}
+        />
+      ),
+      {}
+    );
+  };
+
+  onTabChange = (value: Tab) => {
+    this.setState({tab: value});
+  };
+
+  featureTags() {
+    return this.plugin.features.map(feature => (
+      <StyledTag key={feature}>{feature.replace(/-/g, ' ')}</StyledTag>
+    ));
+  }
+
+  mapPluginToProvider() {
+    const plugin = this.plugin;
+    return {
+      key: plugin.slug,
+    };
+  }
+
+  getTabDiplay(tab: Tab) {
+    //we want to show project configurations to make it more clear
+    if (tab === 'configurations') {
+      return 'project configurations';
+    }
+    return tab;
+  }
```

Comment:
This function, or where it's implemented should be surrounded by the `t()` function, right?

Reply:
The raw strings are always the things that should be wrapped in `t()`, using a variable in `t` defeats it, since we can't map raw strings to translations then

Reply:
@leeandher Good catch!

Reply:
@EvanPurkhiser Is that true? What if we have translations for the different values of the input of `t()`?